ml_peg/app/utils/register_callbacks.py

```diff
@@ -29,18 +30,21 @@ def register_summary_table_callbacks() -> None:
     @callback(
         Output("summary-table", "data"),
         Output("summary-table", "style_data_conditional"),
+        Output("summary-table-computed-store", "data"),
         Input("all-tabs", "value"),
         Input("summary-table-weight-store", "data"),
+        Input("selected-models-store", "data"),
         State("summary-table-scores-store", "data"),
-        State("summary-table", "data"),
+        State("summary-table-computed-store", "data"),
         prevent_initial_call=False,
     )
     def update_summary_table(
         tabs_value: str,
-        stored_weights: dict[str, float],
-        stored_scores: dict[str, dict[str, float]],
-        summary_data: list[dict],
-    ) -> list[dict]:
+        stored_weights: dict[str, float] | None,
+        selected_models: list[str] | None,
+        stored_scores: dict[str, dict[str, float]] | None,
+        summary_data: list[dict] | None,
+    ) -> tuple[list[dict], list[dict], list[dict]]:
         """
         Update summary table when scores/weights change, and sync on tab change.
```

Comment:
Worth updating this too. If it's too long, you can do an extended description e.g.

```
Summary description.

Extended description.

Parameters
-----------
...
```